### ISSUE TITLE
Real typing (remove any types)

### ISSUE THREAD

**Issue by JonEsparaz** [jesus-collective/mobile#289]
Bugs like #287 have made me weary of bad typing. I'll chip away at this over the next few weeks.

**Comment by GeorgeBellTMH** [comment]
Yup...bad habits from my early days of coding in languages with no typing at all....

**Comment by lucastbelem** [comment]
@GeorgeBellTMH @itbel did we get this done?
